test_app/tests/resource_registry/test_resources_api_rest_client.py

```diff
@@ -0,0 +1,158 @@
+import uuid
+
+import pytest
+from crum import impersonate
+from django.conf import settings
+from django.contrib.auth import get_user_model
+
+from ansible_base.resource_registry.models import Resource, service_id
+from ansible_base.resource_registry.rest_client import ResourceAPIClient
+
+
+@pytest.fixture
+def system_user():
```

Comment:
Does the one in `conftest.py` not work in this case? If not, can we fix that one and not duplicate?

Reply:
oh, i didn't even realize we had one. I think I need the `with impersonate`, but I'll try it.